### ISSUE TITLE
remap keyboard shortcuts doesn't work for `quick add`

### ISSUE THREAD

**Issue by Jackiexiao** [remnoteio/remnote-issues#1318]
after setting, the keyboard shortcuts for quickadd is still `ctrl shift alt \``
![image](https://user-images.githubusercontent.com/18050469/133883063-08e23e01-a510-4dd0-88fe-4cb992ff0258.png)

**Comment by hannesfrank** [comment]
Yeah, these require some additional implementation we just did not have the manpower to do yet.

Duplicate of #377

**Comment by Jackiexiao** [comment]
thx for your answer

**Comment by Jackiexiao** [comment]
I let this issue open, and hopefully it could be solved in the future

it would be great that not just made the icon `grey` but also unclickable, so it wouldn't make user confused
![image](https://user-images.githubusercontent.com/18050469/133891175-710d18b2-b58f-43c7-b87c-7d9545f55b55.png)
